doc/lua_api.txt

```diff
@@ -8680,10 +8685,22 @@ Used by `minetest.add_particle`.
 
 `ParticleSpawner` definition
```

Comment:
In the section for individual particles (above here), is `drag` missing? I saw it in some code and am wondering if you added support for it to add_particle too.

Reply:
i don't think i did, i'll see about adding that

Reply:
`drag` and `bounce` can now be set from `add_particle`